WordPress/gutenberg
# Make save panel a dialog with proper labels, fix site editor focus loss after save

## What?


* Makes the save panel a semantic dialog.
* Prevents focus loss after save panel closes in the site editor. A follow-up is being worked on for the post editor but that will be slightly more challenging.

## Why?


Fix accessibility.

## How?


See code.

## Testing Instructions





1. Open the site editor and select Edit.
2. Make a change.
3. Notice how the Open save panel button now tells users it opens a dialog.
4. Select Open save panel.
5. Notice how the save panel is now rendered as a dialog with proper `aria-labelledby` and `aria-describedby` attributes matching the proper IDs.
6. Notice how pressing Save or Cancel will close the save panel and focus loss will no longer occur.

### Testing Instructions for Keyboard


## Screenshots or screencast
